Tests/ArcGISToolkitTests/PreplannedMapModelTests.swift

```diff
@@ -270,7 +270,9 @@ class PreplannedMapModelTests: XCTestCase {
         ]
         
         for (status, expected) in zip(statuses, expected) {
-            status.assertExpectedValue(expected)
+            if status != expected {
+                XCTFail("Status '\(status)' does not match expected status of '\(expected)'")
+            }
         }
```

Comment:
This (and what comes before) is no longer needed. After the sleep, you just need one assertion now:

```swift
XCTAssertEqual(
    statuses
    [.notLoaded, ...]
)
```